assets/scss/03-organisms/_header-hamburger.scss

```diff
@@ -1,474 +1,739 @@
+
+    @media ($bp-small-max) {
+      display: none;
+    }
+
+    @media ($bp-small-min) {
+      width: 66.12903%; //@include span-columns(8);
+    }
+
+    @media ($bp-header-logo-min) {
+      width: 74.59677%; //@include span-columns(9);
+    }
+
+    @media ($bp-small-max) {
+
+      .ma__header-search {
+
+        &__input {
+          display: none;
+        }
+
+        span {
+          display: inline-block;
+        }
+
+        // not the button in navigation
+
+        .ma__button-search {
+
+          &,
+          &--secondary {
+
+            display: none;
+          }
+        }
+      }
+    }
+  }
+
+  &__nav-container {
+    background-color: $c-gray-lightest;
+    height: 95vh;
+    top: 40px;
+    overflow-y: auto;
+    position: absolute;//fixed;
+    left: -320px;
+    // visibility for keyboard and screen reader users are managed with toggling aria-hidden value with js.
+    visibility: hidden;
+    margin-top: 0;
+    transition: left 0.3s ease, visibility 0.3s;
+    width: 320px;
+    z-index: $z-dropdown;
+
+    @media ($bp-header-toggle-min) {
+
+      top: 43px;
+    }
+
+    .show-menu & {
+      visibility: visible;
+      left: 0;
+      transition: left 0.6s ease, visibility 0.8s;
+    }
+
+    .translated-rtl &,
+    .translated-ltr & {
+      height: calc(100vh - #{$header-mobile-controls-height * 2});
+    }
+
+    .translated-rtl &,
+    .translated-ltr & {
+      top: 40px;
+    }
+  }
+
+  &__menu-button {
+    background-color: $c-primary;
+    text-align: left;
+    border: none;
+    font-size: 19.2px;//1.2rem; Maintain the same font size regardless of the screensize.
+    font-weight: 700;
+    color: $c-font-inverse;
+    height: 40px;
+    line-height: 35px;
+    margin-top: 0;
+    padding-right: 30px;
+    padding-left: 20px;
+    transition: transform 0.5s ease;
+
+    span:first-child {
+      opacity: 0.7;
+    }
+
+    @media ($bp-medium-min) {
+
+      padding-left: 30px;
+    }
+
+    @media ($bp-header-toggle-min) {
+
+      padding-left: 0;
+    }
+
+  }
+
+  &__menu-icon {
+    border-radius: 1px;
+    cursor: pointer;
+    background-color: $c-font-inverse;
+    display: block;
+    height: 2px;
+    position: absolute;
+    transition: all 500ms ease-in-out;
+    width: 15px;
+    top: 19px;
+
+    &:before,
+    &:after {
+      cursor: pointer;
+      content: '';
+      background-color: $c-font-inverse;
+      display: block;
+      height: 2px;
+      position: absolute;
+      right: 0;
+      transition: all 500ms ease-in-out;
+      width: 15px;
+    }
+
+    .show-menu & {
+      background-color: transparent;
+    }
+
+    &:before {
+      top: -5px;
+
+      .show-menu & {
+        top: 0;
+        transform: rotate(45deg);
+      }
+    }
+
+    &:after {
+      bottom: -5px;
+
+      .show-menu & {
+        top: 0;
+        transform: rotate(-45deg);
+      }
+    }
+  }
+
+  &__menu-text {
+    position: relative;
+    display: inline-block;
+    // top: -2px;
```

Comment:
Let's delete this comment if it's not needed

Reply:
The comment is removed.